compiler/ballerina-parser/src/main/resources/syntax_diagnostic_message.properties

```
@@ -313,3 +313,5 @@ error.invalid.expression.expected.a.call.expression=invalid expression, expected
 error.metadata.not.attached.to.a.top.level.construct=metadata not attached to a top level construct
 error.metadata.not.attached.to.a.object.member=metadata not attached to a object member
 error.type.desc.after.rest.descriptor=type descriptor not allowed after rest descriptor
+error.configurable.var.implicitly.final='final' qualifier not allowed: configurable variables are implicitly final and \
+                                                                                                             readonly
```

Shall we say only 
```suggestion
error.configurable.var.implicitly.final='final' qualifier not allowed: configurable variables are implicitly final
```

Because the final qualifier doesn't imply read-onlyness.

---

The left side is the reason right?. I think it's okay to keep the first error msg